src/fonts/ionicons.scss

```diff
@@ -33,3 +33,20 @@ ion-icon {
   text-transform: none;
   speak: none;
 }
+
+@mixin scale-x($amount) {
```

Comment:
@manucorporat can you please explain a bit more? If I only do `transform: scaleX(-1);` it does not cover browsers support, so I am not sure what you mean.